modules/op-tkn-hub-interaction.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// *  cli_reference/tkn_cli/op-tkn-reference.adoc
+
+[id="op-tkn-hub-interaction_{context}"]
+= Hub interaction commands
+
+Interact with Tekton Hub for resources such as tasks and pipelines.
+
+== hub
+Interact with hub.
+
+.Example: Display help
+[source,terminal]
+----
+$ tkn hub -h
+----
+
+.Example: Interact with hub API server
+[source,terminal]
+----
+$ tkn hub --api-server https://api.hub.tekton.dev
+----
+
+[NOTE]
+====
+For each command examples given below, you can get the corresponding sub-commands and flags by executing `tkn hub <command> --help`.
```

Comment:
```suggestion
For each example, to get the corresponding sub-commands and flags, run `tkn hub <command> --help`.
```
IBM Style Guide: 
"execute verb
Use only if a simpler term, such as "run", "issue", "start", or "enter" is inappropriate."

Reply:
IBM SG:
**below** adjective, adverb, preposition
Do not use to refer to an earlier software version or fix level. Use "earlier". Do not use to indicate a relative
location in a document, as in "the information below". Use "the information later in this section". Alternatively,
you can use "following" to refer to a table, to art, or to a list. Example: "The following table shows ..."